test/test_ops.py

```diff
@@ -462,9 +471,10 @@ def test_boxes_shape(self):
 
     @pytest.mark.parametrize("aligned", (True, False))
     @pytest.mark.parametrize("device", cpu_and_cuda_and_mps())
-    @pytest.mark.parametrize("x_dtype", (torch.float16, torch.float32, torch.float64), ids=str)
+    @pytest.mark.parametrize("x_dtype", (torch.float16, torch.float32, torch.float64))  # , ids=str)
```

Comment:
Had to comment-out the `ids` part because we're hitting this assert :thinking:  https://github.com/pytorch/pytorch/blob/2a40b7efcb273d1689b55763fade49263adcc788/torch/testing/_internal/optests/generate_tests.py#L225